tgui/packages/tgui/routes.tsx

```diff
@@ -16,8 +16,8 @@ const requireInterface = require.context('./interfaces');
 const routingError =
   (type: 'notFound' | 'missingExport', name: string) => () => {
     return (
-      <Window>
```

Comment:
Don't touch this file.

Reply:
Why? It locks people out of UIs the way it is. If it's used for non-computer UIs or otherwise incompatible in certain situations, then what's the solution for locking people out of their UIs when they reach this screen?

Reply:
For clarity, this is what the pages looks like currently- it locks your device. Closing and reopening it shows the same page, with no way to navigate back to the programs page. The only fix is the pAI owner wiping the pAI and redownloading.

![image](https://github.com/Aurorastation/Aurora.3/assets/89751433/e7e52f9c-439c-4b1c-95f1-9596da56a947)


With NtosWindow:

![image](https://github.com/Aurorastation/Aurora.3/assets/89751433/29441205-ec76-443c-879e-b8cad545a2b9)


Given, the other changes to hide programs without IDs stops this from being a problem for pAIs, I was just trying to be proactive to prevent it from happening in the future, or with the missingExport flag.

Reply:
Because that file is for all UIs - we don't want all UIs to be NTOSWindows. 

The problem you're describing above is just that the pAI Flashlight shouldn't open at all because it's a service - you shouldn't be editing general functionality to fix it.

Reply:
I was not editing general UI functionality for the pAI flashlight - I was editing it for other places where the same might occur and lock a device.

Ideally we would add proper routing for NtosWindow error pages, but as this PR is pAI focused and the problem is solved another way, I'll leave this page be and track the issue in a bug report.